src/Components/HorizontalHandle/HorizontalHandle.js

```
@@ -0,0 +1,71 @@
+        e.preventDefault();
+        e.stopPropagation();
+
+        document.addEventListener("mousemove", handleMouseMove);
+        document.addEventListener("mouseup", handleMouseUp);
+
+        downValueX.current = e.clientX;
+        leftWidth.current = leftDiv.current.getBoundingClientRect().width;
+        rightWidth.current = rightDiv.current.getBoundingClientRect().width;
+
+        handleRef.current.classList.add("handle-active");
+    }, [handleMouseMove, handleMouseUp]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)